meta-mender-core/recipes-mender/mender-client/mender-client-cpp_git.bb

```diff
@@ -0,0 +1,14 @@
+require mender-client-cpp.inc
+
+LICENSE = "Apache-2.0 & BSD-3-Clause"
+
+LIC_FILES_CHKSUM = " \
+    file://LICENSE;md5=4cd0c347af5bce5ccf3b3d5439a2ea87 \
+    file://vendor/googletest/LICENSE;md5=cbbd27594afd089daa160d3a16dd515a \
+    file://vendor/json/LICENSE.MIT;md5=f969127d7b7ed0a8a63c2bbeae002588 \
+    file://vendor/json/docs/mkdocs/docs/home/license.md;md5=970ea048f6ea7d04eeb3ba3ef9ebca40 \
+"
+
+SRCREV = "${AUTOREV}"
+
+SRC_URI = "gitsm://github.com/oleorhagen/mender;protocol=https;branch=yocto-build-branch"
```

Comment:
Obviously not the final branch

Reply:
Yeah I just ran into that too. Wouldn't it be better to have a preliminary branch on the `mendersoftware/mender` repo and point to that instead of referencing a personal repo?

Reply:
Before it's merged, does it matter?